docs/workforce-management-framework/topics/pro-development-and-contributing.adoc

```diff
@@ -0,0 +1,30 @@
+[id='{context}-pro-development-and-contributing']
+= Developing and Contributing to the {PROJECT_NAME} Project
```

Comment:
There will be huge confusion what development means as there are 3 categories of development:
- Core development (contributing)
- RainCatcher step development (plugins,modules)
- Development on the mobile,portal,server application

This confuses even myself and  I think we need to have clear separation between contribution, development of the steps and adding custom functionalities. 

With this in mind `Developing and Contributing to the {PROJECT_NAME} Project` => `Contributing to the {PROJECT_NAME} Project`

Reply:
Good insight, I'll try to update and match these categories. :+1: